_posts/2016-06-27-installing-xhprof.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Installing XHProf to profile your Drupal module"
+date: 2016-05-27
+author: Tim Stallmann
+tags: php drupal module-development
+summary: Step-by-step, how to install and use XHProf to profile your Drupal module.
+---
+
+I had run across a number of blog posts about using XHProf + Drupal, but never actually got it to work sucessfully for a project. Because so much of the documentation online is incomplete or out-of-date, I thought it would be useful to document my process using XHProf to profile a Drupal 8 custom module here. YMMV, but please post your thoughts/experiences in the comments!
```

Comment:
@timstallmann it's clear your audience is for folks who know what XHProf is and are trying to find some documentation. However, for those that don't fit that, some context as to what it does, and why it's good would be helpful up front. I think that can be taken care of in a 3 or 4 sentence intro.